lib/service/cfg.go

```diff
@@ -960,6 +963,26 @@ type MetricsConfig struct {
 	GRPCClientLatency bool
 }
 
+// TracingConfig specifies the configuration for the tracing service
+type TracingConfig struct {
+	// Enabled turns the tracing service role on or off for this process.
+	Enabled bool
+
+	// ExporterURL is the OTLP exporter URL to send spans to.
+	ExporterURL string
+
+	// KeyPairs are the key and certificate pairs that the tracing service will.
+	// use for mTLS.
```

Comment:
```suggestion
	// KeyPairs are the paths for key and certificate pairs that the tracing service will use for outbound TLS connections.
```

Reply:
Should we ask Cloud if they'd like a way to specify these inline in the configuration file?